backends/graphics/sdl/sdl-graphics.cpp

```diff
@@ -558,13 +562,15 @@ void SdlGraphicsManager::initImGui(SDL_Renderer *renderer, void *glContext) {
 	}
 	ImGuiIO &io = ImGui::GetIO();
 	io.ConfigFlags |= ImGuiConfigFlags_DockingEnable;         // Enable Docking
+	io.ConfigFlags |= ImGuiConfigFlags_NoMouse;               // Disable mouse until required
```

Comment:
Do we really want to do this?

Reply:
**Update**: With this change, ImGui capturing of mouse events is disabled by default, which is good :)
Engines enable ImGui mouse events in the `onImGuiRender()` methods, where we do something like this:

    ImGui::GetIO().ConfigFlags |= ImGuiConfigFlags_NoMouseCursorChange | ImGuiConfigFlags_NoMouse;